modal/__init__.py

```diff
@@ -9,16 +9,16 @@
 from modal_version import __version__
 
 try:
-
     from ._container_io_manager import interact, is_local
     from ._tunnel import Tunnel, forward
+    from ._utils.function_utils import current_function_call_id, current_input_id
```

Comment:
Feels a little weird to be importing functions from a private subpackage into the top-level namespace IMO.